stock_picking_backorder_strategy/__manifest__.py

```diff
@@ -4,7 +4,7 @@
 
 {
     'name': 'Picking backordering strategies',
-    'version': '10.0.0.1.0',
+    'version': '10.0.0.1.1',
     'author': 'ACSONE SA/NV, Odoo Community Association (OCA)',
     'category': 'Warehouse Management',
     'website': 'http://www.acsone.eu',
```

Comment:
URL should be `https://github.com/OCA/stock-logistics-workflow`

Reply:
@Nikul-Chaudhary -bot
This is a fix, the module was in 10.0 before. This could be an improvement.

Reply:
@rousseldenis In v10  [here](https://github.com/OCA/stock-logistics-workflow/blob/10.0/stock_picking_backorder_strategy/__manifest__.py#L10), it's still `http://www.acsone.eu`